ps-alias.ps1

```diff
@@ -273,6 +273,28 @@ function Invoke-GitBinding {
     'f' { git fetch $Params }
     # init
     'i' { git init $Params }
+     # ignore
+     'ig' { 
+        try {
+            # Move up the directory tree until we find the .git directory.
+            while ($null -eq (Get-Item -Path .git -Force -ErrorAction SilentlyContinue)) {
+              if((Get-Item .).FullName -eq (Get-Item .).Root) {
+                throw
+              }
+                Push-Location ..
+            }
+            $Params | Add-Content .gitignore 
```

Comment:
Because `$Params` is an array of Strings, this line will add the parameters to the `.gitignore` file each on their own line.

My quick and dirty test:

```powershell
C:\Users\corbob> function g {
>> Param(
>> [Parameter(Mandatory = $true, Position = 0)]
>>     [String]
>>     $Cmd,
>>
>>     [Parameter(Mandatory = $false, ValueFromRemainingArguments = $true)]
>>     [String[]]
>>     $Params
>> )
>> $params | Add-Content .\test.txt
>> }
C:\Users\corbob> g lkj jkl lll jjj kkk sdf
C:\Users\corbob> cat .\test.txt
jkl
lll
jjj
kkk
sdf
```